lib/compat/wordpress-6.1/block-template-utils.php

```diff
@@ -121,6 +121,13 @@ function gutenberg_get_block_templates( $query = array(), $template_type = 'wp_t
 			continue;
 		}
 
+		if ( $post_type &&
+			isset( $template->post_types ) &&
+			! in_array( $post_type, $template->post_types, true )
+		) {
+			continue;
+		}
```

Comment:
@Aljullu, this check is needed for skipping. I've not yet tested your core patch (https://github.com/WordPress/wordpress-develop/pull/3131), but it might be good to double-check.

Reply:
Ugh, sorry I missed that 😅